pygerduty/v2.py

```diff
@@ -71,7 +71,27 @@ def create(self, **kwargs):
         if "requester_id" in kwargs:
             extra_headers["From"] = kwargs.pop("requester_id")
 
-        data[self.sname] = kwargs
+        new_kwargs = {}
+
+        for kwarg_key, kwarg_value in kwargs.iteritems():
+            if kwarg_key.endswith('_id'):
+                new_key = kwarg_key[:-3]
+                new_kwargs[new_key] = {
+                    'id': kwarg_value,
+                    'type': new_key
+                }
+            elif kwarg_key.endswith('_ids'):
+                new_key = kwarg_key[:-3]
```

Comment:
I believe we need to go from `foo_ids` to `foos`. This will go from `foo_ids` to `foo_`. Additionally we should be able to reuse some of this logic for `ids` from `id`. Would you mind moving this two functions `id_to_obj` and `ids_to_objs`, the latter making use of the former, and adding tests to verify the translations?